CHANGELOG.md

```diff
@@ -19,7 +19,9 @@
         - Improve layout of some admin pages.
     - Security:
         - Increase minimum password length to eight.
-
+* v3.1.1 (4th February 2021)
```

Comment:
We don't give a version number/date until release, new rows should be added to the "Unreleased" section above; this would go under Front end improvements, I think, not Development (which is for internal non-user facing stuff).